chef/cookbooks/crowbar/templates/default/crowbar-post-upgrade.sh.erb

```diff
@@ -53,12 +53,13 @@ fi
 
 if [[ -f /sbin/drbdadm ]] ; then
     # Wait until DRBD is started
-    while crm resource status ms-drbd-{postgresql,rabbitmq} | grep "NOT running"; do
+    while [[ $(drbdadm status | wc -c) -eq 0 ]]; do
         sleep 1
```

Comment:
1. It actually logs the command command (see `set -x` at start of the script) .. although extra echo might help.

2. timeout is there, see the execution if the script via post-upgrade methid in api/node